docs-ref-mapping/reference-latest.yml

```diff
@@ -265,11 +265,6 @@
         landingPageType: Service
         children:
         - com.microsoft.azure.documentdb.bulk*
-      - name: Sync
```

Comment:
Please remove from `reference-unified.yml`. This change will be blown away by the first CI run unfortunately :(

Reply:
@scbedd - thanks for letting me know, this is done now, pls check.